build.sbt

```diff
@@ -16,3 +14,8 @@ libraryDependencies += "org.awaitility" % "awaitility" % "2.0.0" % Test
 libraryDependencies += "org.assertj" % "assertj-core" % "3.6.2" % Test
 libraryDependencies += "org.mockito" % "mockito-core" % "2.1.0" % Test
 testOptions in Test += Tests.Argument(TestFrameworks.JUnit, "-a", "-v")
+
+javacOptions ++= Seq(
```

Comment:
Also add `-Werror`?

Reply:
Oh, SGTM. We'll need to adapt other samples too.